### ISSUE TITLE
Make neural net hyperparameters controllable

### ISSUE THREAD

**Issue by ikmckenz** [ikmckenz/target-pred-py#27]
Make the neural net hyperparameters like epochs, learning rate, momentum, batch size, etc., controllable by class parameters.
